### PR TITLE
Updates brave-ui to 0.24.9, adds exclude locale

### PR DESCRIPTION
**Pull request by ryanml**
Fixes: https://github.com/brave/brave-browser/issues/1257

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Needed or QA/No-QA-Needed) to include the closed issue in milestone 

## Test Plan:
 1. Build Brave from this branch (be sure to npm install)
 2. Enable rewards
 3. Hover over settings icon on brave auto-contribute
 4. Ensure that tooltip appears

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

### CODE REVIEW

File: components/resources/brave_components_resources.grd [hunk 1]
```diff
@@ -266,6 +266,7 @@
       <message name="IDS_BRAVE_UI_CLOSE_BALANCE" desc="">Closing Balance</message>
       <message name="IDS_BRAVE_UI_CONTRIBUTE" desc="">Contribute</message>
       <message name="IDS_BRAVE_UI_CONTRIBUTE_ALLOCATION" desc="">Auto-Contribute Allocation</message>
+      <message name="IDS_BRAVE_UI_CONTRIBUTE_TOOLTIP" desc="">Brave Auto-Contribute Settings</message>
```
Comment (NejcZdovc) [inline components/resources/brave_components_resources.grd]: Needs to be just `Auto-Contribute Settings`

Comment (ryanml) [reply components/resources/brave_components_resources.grd]: It was my understanding that auto-contribute references were to be named: `Brave Auto-Contribute` (issue reference https://github.com/brave/brave-core/pull/457)

cc: @jenn-rhim 

If this is the change that needs to be made, should I update UI as well? We shouldn't have differences in locales between ui and here.

Comment (NejcZdovc) [reply components/resources/brave_components_resources.grd]: That was changed yesterday. You can see that we only have Auto-Contribute now (`IDS_BRAVE_UI_CONTRIBUTE_ALLOCATION`)

Comment (NejcZdovc) [reply components/resources/brave_components_resources.grd]: https://github.com/brave/brave-core/pull/502

**Comment by jenn-rhim** [comment]
@ryanml confirming that all occurrences of 'Brave Auto-Contribute' string should change to 'Auto-Contribute' without Brave.

**Comment by NejcZdovc** [comment]
master 74dbfbd
0.56.x 5a8059fc
0.55.x aa1164ec

**Pull request merged**